### ISSUE TITLE
Do not open Gephi in my pc

### ISSUE THREAD

**Issue by dennyseduardo** [gephi/gephi#981]
Hi
I'm having problems to open the Gephi in my computer. I've installed and I tried to open the software. But show the splash screen and after nothing happens. 
I did installed in another computer, but I'd get the same error. 

What I can do to solve this problem?

Thanks

Dennys Eduardo Rossetto

**Comment by yasamans** [comment]
Dear patricia.aparecida

You need Java Development Kit (JDK) 1.7 to run Gephi, I had version 1.8 but it did not work, I installed 1.7 and I have Gephi running perfectly on windows 8 (x64) with Java 1.7 (x64).
The good news is that you do not need to uninstall Java 1.8, you can install Java 1.7 at the same time.

Instructions:
1. Head over to http://www.oracle.com/technetwork/java/javase/downloads/jdk7-downloads-1880260.html and install Java 1.7
2. Go to your Gephi installation folder (probably C:\Program Files (x86)\Gephi-0.8.2) and locate the folder etc
3. Inside etc folder you will find a file named gephi.config, open this file with notepad.
4. Search for the keyword #jdkhome="/path/to/jdk"
5. Once you find this part of code, remove the # from the beginning since the machine will not execute this code and it will consider it as comment.
6. Replace the text inside double quotations /path/to/jdk with the directory address of your Java folder. In order to find your Java folder, check your windows drive (probably C:), and go to Program Files, and find the folder Java, inside that folder you will find a folder that starts with "jdk1.7". You need to copy the address of this folder and paste it instead of /path/to/jdk in the gephi.config file.
For me it was C:\Program Files\Java\jdk1.7.0_67.
7. Save the config file, if you are not allowed to do so save it on some other place and once done, replace it on the original file.

So to sum up, remove the # from gephi.config, replace the JDK path with your Java1.7 folder, and you're done!

Let me know if you have any questions, and please let me know if I can make these instructions any better in any way since I'd wish for it to be the ultimate solution for everyone, I had to run around in the forum to figure what I have to do.

**Comment by eduramiba** [comment]
Duplicate of #748

Gephi does not work in Mac OS since Java 7 and Windows since Java 8.
It's due to problems with old JOGL version and is going to be updated in next version.

Please check https://forum.gephi.org/viewtopic.php?f=3&t=3580&p=10712#p10712 for help about running Gephi with Java 6.
